cognite_toolkit/_cdf_tk/commands/pull.py

```diff
@@ -627,6 +640,13 @@ def _pull_build_dir(
             build_dir, read_resource_folders=built.read_resource_folders, include=None
         )
 
+        if len(selected_loaders) == 0:
+            if isinstance(selected, Path):
+                self.warn(LowSeverityWarning(f"No valid resource recognized at {selected}"))
```

Comment:
I guess you want to do this?.
```suggestion
                self.warn(LowSeverityWarning(f"No valid resource recognized at {selected.as_posix()}"))
```